test/load-suite.js

```diff
@@ -1,17 +0,0 @@
-const setup = require( 'setup' );
-
-function requireTestFiles( config, path = '' ) {
-	Object.keys( config ).forEach( ( folderName ) => {
-		const folderConfig = config[ folderName ];
-
-		if ( folderName === 'test' && Array.isArray( folderConfig ) ) {
-			folderConfig.forEach( fileName => require( `${path}test/${fileName}` ) );
-		} else {
-			describe( folderName, () => {
```

Comment:
>Are there any reason to keep tree like structure?

I most like it in that it helps clarify how parts of the application relate to each other by their folder hierarchy, but I don't think it's absolutely necessary.